src/integrate.ts

```diff
@@ -0,0 +1,43 @@
+import {DopriControlParam} from "./control";
```

Comment:
The idea here is that the top-level `integrate` function just works provided that the two functions agree in whether they need delay information or not.  I based the approach here off of https://www.typescriptlang.org/docs/handbook/advanced-types.html - does this seem like a reasonable way to do it?

Reply:
This approach makes sense to me, yes.